cli/JSXV4UPGRADE.md

```diff
@@ -0,0 +1,43 @@
+**JSX V4 upgrade path**
+
+1. No need to upgrade rescript-react or change bsconfig.json
+
+   JSX V3 by default.
+
+2. New opt-in configuration added in bsconfig.json
+
+   a. JSX V3
+
+   ```json
+   "jsx": {
+     "version": 3,
+     "module": "react",
+     "mode": "classic"
```

Comment:
why even mention `mode` here?
and why even mention `react` as V3 will only work with react?

Reply:
Thanks. removed.